src/main/resources/application.properties

```diff
@@ -1,35 +1,27 @@
 # Info
-info.application.artifactId=@project.artifactId@
-info.application.version=@project.version@
-info.application.description=@project.description@
-info.properties.environment=${ENV:azure}
-
+application.name=${name}
+build.version=${version}
 # Actuator
 management.endpoints.web.exposure.include=health,info
 management.endpoints.jmx.exposure.include=health,info
 management.info.env.enabled=true
 management.endpoint.health.probes.enabled=true
 management.health.livenessState.enabled=true
 management.health.readinessState.enabled=true
-
 # Server
 server.servlet.context-path=/
 server.port=8080
-
 # Logging
-logging.level.root=${DEFAULT_LOGGING_LEVEL:INFO}
-logging.level.it.pagopa=${APP_LOGGING_LEVEL:INFO}
-logging.level.web=${WEB_LOGGING_LEVEL:debug}
-
+logging.level.root=\${DEFAULT_LOGGING_LEVEL:INFO}
+logging.level.it.pagopa=\${APP_LOGGING_LEVEL:INFO}
+logging.level.web=\${WEB_LOGGING_LEVEL:DEBUG}
 # Mongo
-spring.data.mongodb.uri=mongodb://${MONGO_USERNAME}:${MONGO_PASSWORD}@${MONGO_HOST}:${MONGO_PORT}/?ssl=${MONGO_SSL_ENABLED}
+spring.data.mongodb.uri=mongodb://\${MONGO_USERNAME}:\${MONGO_PASSWORD}@\${MONGO_HOST}:\${MONGO_PORT}/?ssl=\${MONGO_SSL_ENABLED}
```

Comment:
What's the difference between these properties (with `\$`) and the ones with `$` (e.g. line 2)?